openstates/mi/bills.py

```diff
@@ -193,13 +193,27 @@ def scrape_bill(self, chamber, session, bill_id):
                         vote.set_count("yes", len(results["yes"]))
                         vote.set_count("no", len(results["no"]))
                         vote.set_count("other", len(results["other"]))
-
                         for name in results["yes"]:
-                            vote.yes(name)
+                            if len(name.split()) < 5:
+                                vote.yes(name.strip())
+                            if session == "2017-2018":
+                                names = name.split("\t")
+                                for n in names:
+                                    vote.yes(name.strip())
                         for name in results["no"]:
-                            vote.no(name)
+                            if len(name.split()) < 5:
+                                vote.no(name.strip())
+                            if session == "2017-2018":
+                                names = name.split("\t")
+                                for n in names:
+                                    vote.no(name.strip())
                         for name in results["other"]:
-                            vote.vote("other", name)
+                            if len(name.split()) < 5:
```

Comment:
this will need comments to be clear for sure

Reply:
Comment added. It's to prevent sentences like "House Bill No. 4451, entitled" being entered in the voter name column.